driver/parser/check.go

```diff
@@ -90,17 +90,12 @@ func (n *Node) Check(scenario *Scenario) error {
 		errs = append(errs, fmt.Errorf("number of instances must be >= 0, is %d", *n.Instances))
 	}
 
-	if n.Genesis.Import != nil {
-		if err := isGenesisFile(n.Genesis.Import.Path); err != nil {
+	if &n.Genesis != nil {
+		if err := isGenesisFile(n.Genesis.Import, true); err != nil {
```

Comment:
This reads to me like while `genesis` is optional, both `genesis.import` and `genesis.export` must be configured.
It would be nice to allow for case 1. exclusive `genesis.import` only, 2. exclusive `genesis.export` only.

Reply:
I have  also added a check for presence of both import and export, as it seems to be forbidden from you commentary of the `Genesis` struct.